packages/components-library/styleguide.config.js

```diff
@@ -10,7 +10,16 @@ module.exports = {
   sections: [
     {
       name: 'Filters',
-      components: 'src/components/filters/*.vue'
+      sections: [
```

Comment:
Nice to split this into sub-categories! Small side-note: Isn't there a more suitable name for the filters than 'Components'? How about 'Types' or something the like?

Reply:
updated the name to 'Types'